opensrp-reveal/src/main/java/org/smartregister/reveal/viewholder/StructureTaskViewHolder.java

```diff
@@ -25,7 +27,9 @@
  */
 public class StructureTaskViewHolder extends RecyclerView.ViewHolder {
 
-    private static SimpleDateFormat dateFormat = new SimpleDateFormat("M/dd", Locale.getDefault());
+    private final static SimpleDateFormat dateFormat = new SimpleDateFormat("MM/dd", Locale.getDefault());
+    private final static SimpleDateFormat thailandDateFormat = new SimpleDateFormat("dd/MM/yy", Locale.getDefault());
```

Comment:
Rename this variable so that it identifiers the date format and not using the country name